src/main/java/seedu/expense/model/ExpenseBook.java

```diff
@@ -267,8 +270,33 @@ public Amount tallyBalance() {
         return tallyBudgets().subtract(tallyExpenses());
     }
 
+    @Override
+    public String getBudgetBarLabel() {
```

Comment:
2 alternative suggestions:

1) Set predicate in UniqueTagList so that only the category corresponding to the category is left after switch command

2) Add category as CommandResult input